processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -32,6 +32,14 @@ var (
 		featuregate.WithRegisterFromVersion("v0.89.0"),
 		featuregate.WithRegisterReferenceURL("https://github.com/open-telemetry/semantic-conventions/issues/495"),
 	)
+	hostCPUSteppingAsStringID          = "processor.resourcedetection.hostCPUSteppingAsString"
+	hostCPUSteppingAsStringFeatureGate = featuregate.GlobalRegistry().MustRegister(
+		hostCPUSteppingAsStringID,
+		featuregate.StageBeta,
+		featuregate.WithRegisterDescription("Change type of host.cpu.stepping to string."),
+		featuregate.WithRegisterFromVersion("v091.0"),
```

Comment:
As currently written that would be v91 :smile: 
```suggestion
		featuregate.WithRegisterFromVersion("v0.95.0"),
```

Reply:
I was confused :) , `v0.91.0` or `v0.95.0`?

Reply:
The first version changing this will be v0.95.0, so we should put v0.95.0, right?

Reply:
ok, got it!